raiden/tests/scenarios/scenario-funding channel.yaml

```diff
@@ -0,0 +1,256 @@
+            - deposit: {from: 0, to: 1, total_deposit: 69_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 70_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 71_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 72_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 73_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 74_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 75_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 76_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 77_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 78_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 79_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 80_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 81_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 82_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 83_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 84_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 85_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 86_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 87_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 88_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 89_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 90_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 91_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 92_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 93_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 94_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 95_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 96_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 97_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 98_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F9 (Depositing 0 tokens)
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000} ## we expect this to work, right?
```

Comment:
I think this case should throw an error

Reply:
Yes but it didn't in the tests. This might be for the same reason as in issue #2630 ?

Reply:
Yes, probably. So this should be fixed then and should give the right error code.